src/actions/app.js

```diff
@@ -1,7 +1,8 @@
 import axios from 'axios';
-
+import { removeDeprecatedFiltersFromState } from '../util/codemirror-utils';
```

Comment:
I simply used the name that has been used throughout the project. If you want to refactor naming conventions then that is fine, but that isn't within the scope of the PR.